scripts/eachdist.py

```diff
@@ -509,7 +509,7 @@ def lint_args(args):
     )
     runsubprocess(
         args.dry_run,
-        ("isort", "--recursive", ".")
```

Comment:
Just out of curiosity, does the new version of isort does not require this `--recursive` flag or running isort in black mode makes this flag unnecessary?

Reply:
New version works recursively by default and the CLI flag was deprecated.